packages/model/src/messages/calls/CallOfferRequestMessage.ts

```diff
@@ -16,13 +21,31 @@ export class CallOfferRequestMessage extends BaseMessage {
       this.threadId = options.threadId
       this.timestamp = options.timestamp ?? new Date()
       this.connectionId = options.connectionId
+      this.offerExpirationTime = options.offerExpirationTime
+      this.offerStartTime = options.offerStartTime
+      this.description = options.description
       this.parameters = options.parameters
     }
   }
 
   public readonly type = CallOfferRequestMessage.type
   public static readonly type = MessageType.CallOfferRequestMessage
 
+  @Expose()
```

Comment:
Note that these @Expose() decorators are not actually needed, since all fields will be exposed by default.